### ISSUE TITLE
Provide a valid label for form fields 

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/nrrd#2525]
OCIO Violation: Provide a valid label for form fields 

OCIO Note: When on-screen labels are present, they must be programmatically associated with form fields. When on-screen labels are not present, form fields must be given an accessible label. This DIV (role=combobox) does not have a mechanism that allows an accessible name value to be calculated

OCIO priority: 6.9

More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g?e=IXABwa) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/login.php)

Instances

- [ ] Location selection box (2 instances) [Explore Data](https://revenuedata.doi.gov/explore/?dataType=Revenue&location=NF,NA&mapLevel=State&offshoreRegions=false&period=Calendar%20Year&year=2022)
![image](https://user-images.githubusercontent.com/39419003/228363095-19a24232-7b5c-4126-86f0-523827f00561.png)
![image](https://user-images.githubusercontent.com/39419003/228363151-8f655854-7c5f-4707-b24a-ef709d436fcf.png)

- [ ] Data type filter [Explore Data](https://revenuedata.doi.gov/explore/?dataType=Revenue&location=NF,NA&mapLevel=State&offshoreRegions=false&period=Calendar%20Year&year=2022)
![image](https://user-images.githubusercontent.com/39419003/228363095-19a24232-7b5c-4126-86f0-523827f00561.png)
![image](https://user-images.githubusercontent.com/39419003/228363151-8f655854-7c5f-4707-b24a-ef709d436fcf.png)

**Comment by eelzi-ONRR** [comment]
OCIO 2024 Report

Instances: 75
Occurrences: 8%
Severity: 10
Priority: 8.4

**Comment by lpgoldstein** [comment]
No longer relevant
